app/scripts/controllers/network/createJsonRpcClient.js

```diff
@@ -19,11 +21,22 @@ const getTestMiddlewares = () => {
 export default function createJsonRpcClient({ rpcUrl, chainId }) {
   const fetchMiddleware = createFetchMiddleware({ rpcUrl });
   const blockProvider = providerFromMiddleware(fetchMiddleware);
+
+  // create subscription polyfill middleware
```

Comment:
Nit: was this comment added in the wrong place? Was there a reason it was added here but not in the Infura client?

Reply:
Removed here: [e51455a](https://github.com/MetaMask/metamask-extension/pull/16693/commits/e51455a36612ef7a1feb40880863e74c77b0b7f5)